verify.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# This file is part of the Kepler project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 The Kepler Contributors
+#
+
+## basic check for bcc
+if [ $(dpkg -l | grep bcc | wc -l) == 0 ]; then
+    echo "no bcc package found"
+    dpkg -l | grep bcc
```

Comment:
why need to run it again?

Reply:
if there some error happen, we'd better run the script twice to check the status in CI before the CI job exit.
otherwise, there will be no error message printed before exit.

Reply:
let's add `set -x` so we can debug the flow.

Reply:
```
++ wc -l
++ grep bcc
++ dpkg -l
+ '[' 2 == 0 ']'
```
https://github.com/sustainable-computing-io/local-dev-cluster/actions/runs/5156780909/jobs/9288253867?pr=14#step:5:5
as the shell script use pipe to send result into count via `wc -l` ...
I suppose we can have `set -x` for now and let's see.